src/classes/RD2_Settings.cls

```diff
@@ -0,0 +1,162 @@
+* @description Recurring Donation settings wrapper
+*
+*/
+public with sharing class RD2_Settings {
+    
+    /***
+    * @description Recurring Donation custom settings
+    */
+    private npe03__Recurring_Donations_Settings__c settings {
+        get {
+            if (settings == null) {
+                settings = UTIL_CustomSettingsFacade.getRecurringDonationsSettings();
+            } 
+            return settings;
+        }
+        set;
+    }
+
+    /***
+    * @description The installment Opportunity Stage
+    */
+    public String installmentStage {
+        get {
+            if (installmentStage == null) {
+                installmentStage = String.isNotBlank(settings.InstallmentOppStageName__c)
+                    ? settings.InstallmentOppStageName__c
+                    : System.Label.npe03.RecurringDonationStageName;
+            }
+            return installmentStage;
+        } 
+        private set;
+    }
+
+    /***
+    * @description The installment Opportunity Record Type Id
+    */
+    public Id installmentRecordTypeId {
+        get {            
+            return (Id) settings.npe03__Record_Type__c;
+        } 
+        private set;
+    }
+
+    /***
+    * @description Indicates if Campaign should be propagated from RD to the installment Opportunity
+    */
+    public Boolean shouldSetInstallmentCampaign {
+        get {            
+            return settings.npe03__Add_Campaign_to_All_Opportunites__c;
+        } 
+        private set;
+    }
+
+    /***
+    * @description Indicates if open installment Opportunities should be deleted when RD is closed
+    */
+    public Boolean isCloseActionDeletion() {
+        return settings.npe03__Open_Opportunity_Behavior__c == RD2_Constants.CloseActions.Delete_Open_Opportunities.name();
+    }
+
+    /***
+    * @description Indicates if open installment Opportunities should be set to Closed Lost when RD is closed
+    */
+    public Boolean isCloseActionClosedLost() {
+        return settings.npe03__Open_Opportunity_Behavior__c == RD2_Constants.CloseActions.Mark_Opportunities_Closed_Lost.name();
+    }
+
+    /***
+    * @description Indicates if open installment Opportunities should be unchanged when RD is closed
+    */
+    public Boolean isCloseActionNoAction() {
+        return settings.npe03__Open_Opportunity_Behavior__c == RD2_Constants.CloseActions.No_Action.name();
+    }
+
+    /***
+    * @description Indicates if the first installment can be created on a Recurring Donation
+    */
+    public Boolean isFirstInstallmentEnabled {
+        get {
+            if (isFirstInstallmentEnabled == null) {
+                isFirstInstallmentEnabled = true; //Disable First Installment ? false : true;
+            }
+            return isFirstInstallmentEnabled;
+        } 
+        private set;
+    }
+
+    /***
+    * @description Indicates if the next installment can be created on a Recurring Donation
+    */
+    public Boolean isNextInstallmentEnabled {
```

Comment:
Yes, the purpose was to minimize merge conflicts and also ensure this PR covers future changes and it is correct.
The PR #4918 will have this PR #4903 merged in and dev to be continued since the current `feauture/RD2` does not have future Opp update and Opp creation since it is implemented here.